section_7/starter_code/requirements.txt

```diff
@@ -3,5 +3,4 @@ Flask-RESTful
 Flask-JWT
 Flask-SQLAlchemy
 uwsgi
-# need to clarify which dependencies are missing in order to install psycopg2
-psycopg2-binary
+psycopg2
```

Comment:
libpq-dev : Depends: libpq5 (= 12.8-0ubuntu0.20.04.1) but 13.0-1.pgdg18.04+1 is to be installed E: Unable to correct problems, you have held broken packages. 
SOLVED:
libpq5 downgraded to version 12.8